shared/api/network_peer.go

```diff
@@ -61,6 +96,14 @@ type NetworkPeer struct {
 	// Example: network1
 	TargetNetwork string `json:"target_network" yaml:"target_network"`
 
+	// lxdmeta:generate(entities=network-peering; group=peering-properties; key=status)
+	// The only supported keys are `user.*` custom keys.
+	// Indicates if mutual peering exists with the target network.
```

Comment:
Let's state that this is read-only:
"This property is read-only and cannot be updated."

Reply:
fixed